src/main/kotlin/no/nav/familie/ef/mottak/task/SendPåminnelseOmDokumentasjonsbehovTilDittNavTask.kt

```diff
@@ -73,26 +83,31 @@ class SendPåminnelseOmDokumentasjonsbehovTilDittNavTask(
         stønadType: StønadType?,
     ) =
         harSendtInnNySøknad(søknad, søknader) ||
-            harSendtInnEttersendingIEttertid(søknad, ettersendinger) ||
-            stønadType == null ||
-            tilhørendeBehandleSakOppgaveErPåbegynt(søknad, stønadType)
+                harSendtInnEttersendingIEttertid(søknad, ettersendinger) ||
+                stønadType == null ||
+                tilhørendeBehandleSakOppgaveErPåbegynt(søknad, stønadType)
 
     private fun harSendtInnNySøknad(søknad: Søknad, søknader: List<Søknad>): Boolean =
         søknader.filter { it.id != søknad.id }
             .filter { SøknadType.hentSøknadTypeForDokumenttype(it.dokumenttype) != SøknadType.OVERGANGSSTØNAD_ARBEIDSSØKER }
             .any { it.opprettetTid > søknad.opprettetTid }
 
-    private fun harSendtInnEttersendingIEttertid(søknad: Søknad, ettersendinger: List<Ettersending>): Boolean =
-        ettersendinger.any { it.opprettetTid > søknad.opprettetTid }
+    private fun harSendtInnEttersendingIEttertid(søknad: Søknad, ettersendinger: List<Ettersending>): Boolean {
+        dokumentasjonsbehovvarslingerBrukerSendtInn.increment()
```

Comment:
Blir ikke dette feilt? Nå vil den vel inkremmentere verdien for gang man gjør denne sjekken. Må du ikke sjekke på verdien som denne metoden returnerer for å avgjøre om du skal inkremmentere eller ikke?

Reply:
Jo det blir veldig feil, men endrer strategi - kommer til å logge på tre variabler for "ikke sendt" 
bruker, saksbehandler eller begge.